custom-words.txt

```diff
@@ -9,8 +9,9 @@ aapl
 AATP
 abcxyz
 ABFS
-Abstractive
 ABGRABGR
+aborthnsonmigration
+Abstractive
```

Comment:
Is this some special jargon? I think the usual word is not Abstractive but 'abstract' (n., v., adj.), or 'abstracted' (v, p.t.), 'abstractly' (adv.) etc.